core/Cargo.toml

```diff
@@ -43,7 +43,7 @@ serde = { workspace = true }
 serde_json = { version = "1.0", features = ["preserve_order"] }
 nellymoser-rs = { git = "https://github.com/ruffle-rs/nellymoser", rev = "754b1184037aa9952a907107284fb73897e26adc", optional = true }
 regress = "0.10"
-flash-lso = { git = "https://github.com/ruffle-rs/rust-flash-lso", rev = "98679aa0cdcad5fecc5c7321b2eac35b69dbcd5f" }
+flash-lso = { git = "https://github.com/ruffle-rs/rust-flash-lso", rev = "016946cef2392f6b0bf79f79d692880ab76204a5" }
```

Comment:
This is the head of the currently unmerged https://github.com/ruffle-rs/rust-flash-lso/pull/68, which seems to be unfinished? This seems a bit unfortunate...

Would it be possible to update to the current `master` of that repo first?

Reply:
That pr isn't unfinished it's just waiting for review? I don't mind this pr being stalled until that pr is merged, but I'd rather not revert this to a buggier commit

Reply:
I see, sorry then. I interpreted "Note this is not as clean as I would like, but I want to fix the panics first and worry about the refactor later" as you still working on it.